### PR TITLE
CN-6212 loop when session expired

### PR DESCRIPTION
**Pull request by srkela**
## Checklist (Mandatory):

- [x] I have named the PR correctly (Check PR1)
- [x] Check the commit’s or even all commits message styles matches our requested structure. (issue number, etc.) (Check CR10)
- [x] I have tested new code locally (Check CR3)
- [x] I have done a smoke test of the larger feature locally (Check CR3)
- [x] I have added at least two reviewers (Check PR5)
- [x] I have assigned the PR to myself (Check PR4)
- [x] I have merged develop (parent) branch into my feature branch and resolved all conflicts (Check PR7)

## Checklist (Optional):

- [ ] I have added necessary documentation, including instructions how to test (Check CR8)
- [ ] I have added tests that prove my fix/implementation is effective or that my feature works
- [ ] I have written the code according to the adopted style guide? (Check CR7)

## Additional notes

> What does this implement/fix? Explain your changes. (optional)
> Are there any scripts that need running (eg. yarn install)?
> Are there any edge cases that need checking?
> Are there any specific test instructions?

[MVPWorkshop Pull Request & Code Review Guidelines](https://docs.google.com/document/d/1TcuU1JBhlGZLtpy0osSNinXQ2piP_yrDuRcXdqaOS_U/edit#heading=h.5b6kpdr4btq8)

### CODE REVIEW

File: app/components/screens/WalletLanding/WalletLanding.js [hunk 1]
```diff
@@ -109,6 +110,8 @@ class WalletLanding extends Component {
         actions.openModal(MODALS.MULTI_ADDRESS_MODAL);
     }, 2000);
 
+    appUtil.pollBackendStatus();
```
Comment (fjovakaric) [inline app/components/screens/WalletLanding/WalletLanding.js:113]: remove

File: app/utils/api-util.js [hunk 2]
```diff
@@ -251,10 +251,11 @@ async function errorInterceptor(serverError) {
   return Promise.reject(err);
 }
 
-function handle401(err) {
+async function handle401(err) {
   if (err.slug === "SESSION_EXPIRED") {
     store.dispatch(actions.expireSession());
-    store.dispatch(actions.logoutUser());
+    await deleteSecureStoreKey(SECURITY_STORAGE_AUTH_KEY);
```
Comment (fjovakaric) [inline app/utils/api-util.js:257]: move to action logoutFormDevice. reset reducer in `reducers.js`. check current logout action

File: app/utils/app-util.js [hunk 3]
```diff
@@ -107,6 +108,10 @@ async function pollBackendStatus() {
     await store.dispatch(actions.getUserStatus());
     await checkAndRefreshAuthToken();
 
+    if (!isUserLoggedIn()) {
```
Comment (fjovakaric) [inline app/utils/app-util.js:111]: don't clear interval. poll data only when the user is loaded
